docs/index.md

```diff
@@ -0,0 +1,47 @@
+# Song Developer Documentation
+
+---
+
+**Navigation**
+- [Operation](./operation/operation.md)
+- [Contribution](./contribution/contribution.md) 
+
+---
+
+# Background
+
+Song is a metadata validation and tracking tool designed to streamline the management of genomics data across multiple cloud storage systems. With Song, users can create high-quality and reliable metadata repositories with minimal human intervention. As a metadata management system, Song does not handle file transfer and object storage. Song interacts with a required companion application, <a href="https://github.com/overture-stack/score" target="_blank" rel="noopener noreferrer">Score</a>, which manages file transfers and object storage.
+
+## Data Submission
+
+**Analysis Files:** An analysis is a description of a set of one or more files plus the metadata describing that collection of files.
```

Comment:
The label here should just be `**Analyses:**`. We need to help clarify the distinction between an _analysis_ (collection of files and the meta data that describes them), and the _files_ themselves. I think it would help to do that if we separate the terms clearly.